docs/src/pages/components/tables/tables-pt.md

```diff
@@ -79,6 +79,32 @@ Um exemplo de uma tabela com linhas roláveis e cabeçalhos de coluna fixos. It
 
 {{"demo": "pages/components/tables/StickyHeadTable.js", "bg": true}}
 
+##### Nota: Se a tabela tiver múltiplas linhas de cabeçalho, é necessário adicionar estilos adicionais para garantir que todas as linhas fiquem visíveis. Por defeito, as linhas irão acumular uma em cima da outra de forma a que apenas a última linha de cabeçalho ficará visível. Para evitar este comportamento, será necessário estilizar as linhas de cabeçalho adicionais com um valor de `top` para criar o espaçamento necessário.
```

Comment:
We manage the translations in https://translate.material-ui.com/. I have removed the change.

Reply:
Ok, I didn't know that, thanks!